library/std/src/lib.rs

```diff
@@ -212,7 +212,7 @@
 #![needs_panic_runtime]
 // std may use features in a platform-specific way
 #![allow(unused_features)]
-#![cfg_attr(test, feature(internal_output_capture, print_internals, update_panic_count))]
+#![cfg_attr(test, feature(internal_output_capture, print_internals, update_panic_count, rt))]
```

Comment:
necessary because of https://github.com/rust-lang/rust/blob/f5cdb567f12e707331445386177c8c24b88af856/library/std/src/panicking.rs#L408-L409